httputils.nim

```diff
@@ -10,7 +10,7 @@
 {.push raises: [].}
 
 import std/[times, strutils, algorithm, sequtils]
-import results
+import stew/results
```

Comment:
not sure why you closed #36, it was the better PR because it did not have this unrelated results change

Reply:
Because it stayed for too long without any reaction on it, and I assumed that something must be too wrong to be discussed... :)